adp-algorithms/mind/src/main/java/de/metanome/algorithms/mind/Mind.java

```diff
@@ -21,10 +22,12 @@
 
   private final InclusionDependencyInputGenerator inclusionDependencyInputGenerator;
   private final ValidationStrategyFactory validationStrategyFactory;
+  private final InclusionDependencyUtil util;
```

Comment:
<nit> I would initialize objects with no-arg constructors directly when declaring them to unclutter the constructor.

Reply:
These are probably matters of style / taste, but anyway :)
I prefer to initialise fields in the constructor since these structures are typically easier to test. If the field was initialised at the declaration side it would not be possible to write another constructor which swaps the actual impl out for a stub/fake/mock while keeping the field final.

Reply:
Valid point. In this case it might be cleaner to just pass the dependencies as a constructor parameter. However this shouldn't madder right now, especially since your tests are already running.